example-stories-not-used/Header.tsx

```diff
@@ -4,17 +4,27 @@ import { Button } from './Button';
 import './header.css';
 
 interface HeaderProps {
-  user?: {};
+  user?: Record<string, unknown>;
```

Comment:
made the mistake of looking at the file and it complained, so I changed it. oops.